.ci/cico_rhche_prcheck.sh

```diff
@@ -8,7 +8,7 @@
 export PR_CHECK_BUILD="true"
 export BASEDIR=$(pwd)
 export DEV_CLUSTER_URL=https://devtools-dev.ext.devshift.net:8443/
-export OC_VERSION=3.9.33
+export OC_VERSION=3.10.85
```

Comment:
@Katka92 @ScrewTSW what would happen if this `OC_VERSION` will not be set ?

Reply:
tests would fail, because the `oc` binary would not be downloaded.

Reply:
hmm.. so this is the reason of the current failures https://ci.centos.org/view/Devtools/job/devtools-rh-che-rh-che-prcheck-dev.rdu2c.fabric8.io/ ?

Reply:
Not really. Those were caused by the `che-functional-tests` docker image we are using in PR-Check.
Some keycloak token verifying logic is hard-coded in the image. I've temporarily restored the token, so the tests don't fail.

Reply:
ok, please explain why it was not failing before and oc `binary` were downloaded without this env var

Reply:
because the env var was already there in the prcheck script, but with old version. devcluster was updated to 3.10 from 3.9, so I also updated the version of the binary

Reply:
@ScrewTSW should we remove this env var from old prcheck script ? or is it already done ?